source/extensions/filters/network/redis_proxy/proxy_filter.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <bits/types/time_t.h>
```

Comment:
this seems break build on macos m1
```
./source/extensions/filters/network/redis_proxy/proxy_filter.h:3:10: fatal error: 'bits/types/time_t.h' file not found
```

Reply:
Will take a look. Will be hard to repro without a mac, and I but I'm guessing this is not the right import so I'll check what should be the alternative

Reply:
create a CI for this check: https://github.com/envoyproxy/envoy/pull/35933